lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -440,3 +633,30 @@ const Map<String, List<String>> blockedTestsListsMap = <String, List<String>>{
     'target_platform_ios_integration.dart',
   ],
 };
+
+/// Tests blocked for one of the build modes.
```

Comment:
It is more common to call such tests "skipped" rather than "blocked".

Reply:
It is outside of the scope of this work. It has been a while since we added 'blocked' test concept. Blocked word is introduced as a replacement to 'blacklist' a while ago.